src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -54,6 +55,13 @@ public static class WebScriptHostBuilderExtension
                 {
                     configureWebJobs?.Invoke(webJobsBuilder);
 
+                    webJobsBuilder.Services.TryAddSingleton<HttpClient>(f =>
+                    {
+                        var loggerFactory = f.GetService<ILoggerFactory>();
+                        loggerFactory.CreateLogger("Host.Startup").LogWarning("Detected direct usage of HttpClient in function app. Please use HttpClientFactory instead - see <TODO aka.ms link> for more information.");
```

Comment:
Suggesting an updated message: 'Using HttpClient as an injected dependency will not be supported in future versions of Azure Functions. Use IHttpClientFactory instead. See http://aka.ms/functions-httpclient-di for more information.'